lib/buffer.js

```diff
@@ -379,6 +378,16 @@ Buffer.prototype.toString = function(encoding, start, end) {
 };
 
 
+Buffer.prototype.toString = function() {
+  const length = this.length | 0;
```

Comment:
Is the int32 conversion on the length a safeguard in case the length has been altered or `this` isn't actually a Buffer instance?